vispy/visuals/isocurve.py

```diff
@@ -96,8 +106,8 @@ def set_data(self, data):
         if _HAS_MPL:
             try:
                 if not (self._X.T.shape == data.shape):
-                    raise
-            except:
+                    raise Exception
+            except Exception:
```

Comment:
Yeah I *think* that's actually what it's designed to catch, the attribute
not being defined, or as it is now, being None. I'll change the checks to
be explicit as I agree it's much clearer that way.